samples/quickstart.js

```diff
@@ -17,24 +17,51 @@
 // sample-metadata:
 //   title: Quickstart
 //   description: A quick introduction to using the Pub/Sub client library.
-//   usage: node quickstart.js <project-id> <topic-name>
+//   usage: node quickstart.js <project-id> <topic-name> <subscription-name>
+
+function exit(code) {
+  // eslint-disable-next-line no-process-exit
+  process.exit(code);
+}
 
 // [START pubsub_quickstart_create_topic]
 // Imports the Google Cloud client library
 const {PubSub} = require('@google-cloud/pubsub');
 
 async function quickstart(
   projectId = 'your-project-id', // Your Google Cloud Platform project ID
-  topicName = 'my-topic' // Name for the new topic to create
+  topicName = 'my-topic', // Name for the new topic to create
+  subscriptionName = 'my-sub' // Name for the new subscription to create
 ) {
   // Instantiates a client
   const pubsub = new PubSub({projectId});
 
-  // Creates the new topic
+  // Creates a new topic
   const [topic] = await pubsub.createTopic(topicName);
   console.log(`Topic ${topic.name} created.`);
+
+  // Creates a subscription on that new topic
+  const [subscription] = await topic.createSubscription(subscriptionName);
+
+  // Receive callbacks for new messages on the subscription
+  subscription.on('message', message => {
+    console.log('Received message:', message.data.toString());
+    exit(0);
+  });
+
+  // Receive callbacks for errors on the subscription
+  subscription.on('error', error => {
+    console.error('Received error:', error);
+    exit(1);
+  });
+
+  // Send a message to the topic
+  topic.publish(Buffer.from('Test message!'));
```

Comment:
I am still sad that the user has to do this :(  Couldn't we check the typeof the arugment, and based on that convert it to a buffer or not?

Reply:
I like that idea, though I think I'd want to pursue it separate from this change.

Reply:
https://github.com/googleapis/nodejs-pubsub/issues/1127